apps/pip/dev/01/kustomization.yaml

```diff
@@ -1,6 +0,0 @@
-apiVersion: kustomize.config.k8s.io/v1beta1
-kind: Kustomization
-namespace: pip
-
-resources:
-  - ../base
```

Comment:
You might still need this file https://github.com/hmcts/sds-flux-config/blob/8840c05d1bb173707b3f767b7560e9ff37ed5a57/apps/pip/base/kustomize.yaml#L7

Reply:
Thanks have added it back :)